logical/join.go

```diff
@@ -12,10 +12,12 @@ import (
 	"github.com/pkg/errors"
 )
 
+var ErrFallbackToLookupJoin = errors.New("Fallback to lookup join")
```

Comment:
if every other `errors.New` starts with non capital then this one also shouldn't